qvm-tools/qvm-ls

```diff
@@ -24,7 +24,7 @@
 from qubes.qubes import QubesVmCollection
 from qubes.qubes import QubesHost
 from qubes.qubes import QubesException
-from optparse import OptionParser
+from argparse import ArgumentParser
```

Comment:
The Problem with OptionParser is that it doesn't allow the 'nargs="+"'-option. I wanted to have as many fields specified with one command as required. With OptionParser, 'nargs=1' is the only meaningful option => for n fields I need n commands.